db/models.py

```diff
@@ -14,3 +14,41 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return self.name
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(CinemaHall,
+                                    on_delete=models.CASCADE,
+                                    related_name="movie_sessions")
+    movie = models.ForeignKey(Movie, on_delete=models.DO_NOTHING,
+                              related_name="movie_sessions")
+
+    def __str__(self) -> str:
+        return (
+            f"{self.movie.title} "
+            f'{self.show_time.strftime("%Y-%m-%d %H:%M:%S")}'  # noqa: E999
```

Comment:
You can use single quotes inside the f-string, then flake8 will not raise an error.
```suggestion
            f"{self.movie.title} "
            f"{self.show_time.strftime('%Y-%m-%d %H:%M:%S')}"

Reply:
When i use it, i also have flake8 error:
`Q000 Single quotes found but double quotes preferred`